src/js/components/search/collapsibleSidebar/SidebarContent.jsx

```diff
@@ -77,13 +78,22 @@ const SidebarContent = ({
     };
 
     return (<>
-        <SearchSidebarMainMenu
+        {!isDsmOpened && <SearchSidebarMainMenu
             isDrilldown={isDrilldown}
-            sidebarContentHeight={sidebarContentHeight}
+            sidebarContentHeight={sidebarContentHeight - 50}
             setLevel2={setLevel2}
             itemCount={itemCount}
             setShowMobileFilters={setShowMobileFilters} />
-
+        }
+        {currentLevel === 1 && <DsmSlider
+            isDsmOpened={isDsmOpened}
+            setIsDsmOpened={setIsDsmOpened}
+            dsmFile="learn-filters-panel.mdx"
+            currentLevel={1}
+            selectedCategoryTitle=""
+            height={sidebarContentHeight + 50}
```

Comment:
it looks like this is throwing off the calculations for the ds&m slider.  if you scrolldown so only the top blue sticky bar is visible, the "Learn more about filter categories" link gets misaligned - 
![Screenshot 2025-04-09 084019](https://github.com/user-attachments/assets/0d8367f3-bd2e-4a10-8776-caff107e1bc5)